Src/BootCamp.Chapter/PeoplePredicates.cs

```diff
@@ -10,20 +10,21 @@ public static class PeoplePredicates
         /// a) over 18, who do not live in UK, whose surename does not contain letter 'a'.
         /// </summary>
         /// <returns></returns>
-        public static bool IsA(Person person) => false;
+        public static bool IsA(Person person) => person.CalculateAge() > 18 &  person.Country != "UK" & !person.SureName.Contains('a')  ;
```

Comment:
`&&` would be better and should really always be used unless working with bits. Read up on the differences between `&` and `&&`

Reply:
oke, as far as I know && is lazy so when the first is true or false  there is a chance that the rest is not handled.  With &  every case is handled. Or I misunderstood things.

Reply:
After some thought you are right that && is a better choice here. If one of them fails it has no need to look at the conditions after it. the person is already not qualified.

Reply:
@RoelofWobben you did misunderstand things. Both & and && will result in the same true/false. However, && stops immediately after the left side is false. Therefore, in some weird cases, you might want run both left AND right side checks (for example if a check logs something on top of just returning true/false or maybe fires an event, etc).